lib/commands/command-plugin-update

```diff
@@ -10,16 +10,30 @@ plugin_update_command() {
   local gitref="${2:-master}"
   if [ "$plugin_name" = "--all" ]; then
     for dir in "$(asdf_data_dir)"/plugins/*; do
-      echo "Updating $(basename "$dir")..."
-      (cd "$dir" && git fetch -p -u origin "$gitref:$gitref" && git checkout -f "$gitref")
+      local plugin_name
+      plugin_name=$(basename "$dir")
+      update_plugin "$plugin_name" "$dir" "$gitref" &
     done
+    wait
   else
     local plugin_path
     plugin_path="$(get_plugin_path "$plugin_name")"
     check_if_plugin_exists "$plugin_name"
+    update_plugin "$plugin_name" "$plugin_path" "$gitref"
+  fi
+}
+
+update_plugin() {
+  local plugin_name=$1
+  local plugin_path=$2
+  local gitref=$3
+  logfile=$(mktemp)
+  {
     echo "Updating $plugin_name..."
     (cd "$plugin_path" && git fetch -p -u origin "$gitref:$gitref" && git checkout -f "$gitref")
```

Comment:
Would it make sense to pass `-q`/`--quiet` to `git fetch` and `git checkout` to avoid interleaved progress messages from the different commands?